hack/lib/clone_jobs.py

```diff
@@ -0,0 +1,29 @@
+import json, sys, yaml, copy;
+
+def duplicate_job(job, from_branch, to_branch, to_version):
```

Comment:
wait, dupe the ci-op config and use generator?

Reply:
?  I need to branch before end of week and have 100% of jobs work.  we can always tweak later